pom.xml

```diff
@@ -306,44 +306,80 @@
                         <artifactId>qulice-maven-plugin</artifactId>
                          <!--
                          @todo #558:30min Upgrade Qulice to 0.16. Fix all checkstyle
-                          violations (mostly `AtclauseOrderCheck`). Fix or add puzzle for 
-                          PMD, findbugs and other types of violations. 
+                          violations (mostly `AtclauseOrderCheck`). Fix or add puzzle for
+                          PMD, findbugs and other types of violations.
                          -->
                         <version>0.15.4</version>
                         <configuration>
                             <excludes combine.children="append">
                                 <exclude>checkstyle:/src/site/resources/.*</exclude>
                                 <!--
-                                @todo #517:30min Continue fixing FindBugs violations started in #517.
-                                 #517 turned on the FindBugs, temporarily excluding some classes.
+<<<<<<< HEAD
```

Comment:
@sebing looks like something wasn't merged properly.
This line should not be in pom